### PR TITLE
Job automation - adding new server connection types

### PR DESCRIPTION
**Pull request by felix-couprie**
New text paragraphs for each server type

### CODE REVIEW

File: content/md/imports-exports/automate-imports-and-exports.md [hunk 1]
```diff
@@ -12,36 +12,75 @@ related: imports, exports, tailored-export, tailored-import,
...
+1. Click `Save` in the top right corner of the screen
+
+## Connect to Microsoft Azure
+
+To connect your product import/export to Microsoft Azure:
+1. Go to `Imports` or `Exports`
+1. Select the profile you would like to automate
+1. Click `Edit` in the top right corner
+1. Click `Properties`
+1. Under "Type", select `Amazon S3`
```
Comment (StevenVAIDIE) [inline content/md/imports-exports/automate-imports-and-exports.md:69]:
```suggestion
1. Under "Type", select `Microsoft Azure`
```

File: content/md/imports-exports/automate-imports-and-exports.md [hunk 2]
```diff
@@ -12,36 +12,75 @@ related: imports, exports, tailored-export, tailored-import,
...
+
+
+## Connect to Google Cloud Storage
+
+To connect your product import/export to Google Cloud Storage:
+1. Go to `Imports` or `Exports`
+1. Select the profile you would like to automate
+1. Click `Edit` in the top right corner
+1. Click `Properties`
+1. Under "Type", select `Amazon S3`
```
Comment (StevenVAIDIE) [inline content/md/imports-exports/automate-imports-and-exports.md:81]:
```suggestion
1. Under "Type", select `Google Cloud Storage`
```

File: content/md/imports-exports/automate-imports-and-exports.md [hunk 3]
```diff
@@ -12,36 +12,75 @@ related: imports, exports, tailored-export, tailored-import,
...
+1. Fill in all mandatory fields: `Filepath`, `Project ID`, `Service account` and `Bucket`
+1. Click `Save` in the top right corner of the screen
 
 # Schedule and automate your import/export profile
```
Comment (StevenVAIDIE) [inline content/md/imports-exports/automate-imports-and-exports.md:85]: I think we should add a point on the info block: 
- You cannot automate an import without remote storage

**Review by StevenVAIDIE — APPROVED** [review]
I think this change should be also in 7.0 helpcenter documentation so can you create another PR to target v7 branch ?

File: content/md/imports-exports/automate-imports-and-exports.md (hunk 3 above)
Comment (felix-couprie) [reply content/md/imports-exports/automate-imports-and-exports.md:85]: Thanks, added!

**Pull request merged**